lib/views/forget_password/forget_password_view.dart

```diff
@@ -0,0 +1,146 @@
+import 'package:kayla_flutter_ic/views/forget_password/forget_password_state.dart';
+import 'package:kayla_flutter_ic/views/forget_password/forget_password_view_model.dart';
+
+final forgetPasswordViewModelProvider = StateNotifierProvider.autoDispose<
+    ForgetPasswordViewModel, ForgetPasswordState>(
+  (_) => ForgetPasswordViewModel(),
+);
+
+class ForgetPasswordView extends ConsumerStatefulWidget {
+  const ForgetPasswordView({super.key});
+
+  @override
+  ForgetPasswordViewState createState() => ForgetPasswordViewState();
+}
+
+class ForgetPasswordViewState extends ConsumerState<ForgetPasswordView>
+    with TickerProviderStateMixin {
+  AppBar get _appBar => AppBar(
+        leading: const BackButton(),
+        backgroundColor: Colors.transparent,
+        shadowColor: Colors.transparent,
+      );
+
+  Widget get _background => SizedBox(
+        width: MediaQuery.of(context).size.width,
+        height: MediaQuery.of(context).size.height,
+        child: Stack(
+          children: [
+            _backgroundImage,
+            _linearGradientBlurBackground,
+          ],
+        ),
+      );
+
+  Widget get _backgroundImage => Image(
+        image: Assets.images.nimbleBackground.image().image,
+        fit: BoxFit.cover,
+        alignment: Alignment.center,
+      );
+
+  Widget get _linearGradientBlurBackground => Stack(
+        children: [
+          _backgroundImage,
+          ImageFiltered(
+            imageFilter: ImageFilter.blur(sigmaX: 10, sigmaY: 10),
+            child: ShaderMask(
+              shaderCallback: (rectangle) {
+                return LinearGradient(
+                  begin: Alignment.topCenter,
+                  end: Alignment.bottomCenter,
+                  colors: [
+                    Colors.black.withOpacity(0.2),
+                    Colors.black.withOpacity(1),
+                  ],
+                ).createShader(rectangle);
+              },
+              blendMode: BlendMode.overlay,
+              child: _backgroundImage,
+            ),
+          ),
+        ],
+      );
+
+  Widget get _logo => SizedBox(
+        width: 166,
+        height: 40,
+        child: FittedBox(
+          fit: BoxFit.contain,
+          child: Image(
+            image: Assets.images.nimbleLogoWhite.image().image,
+            fit: BoxFit.scaleDown,
+          ),
+        ),
+      );
+
+  Widget _intruction(BuildContext context) => Text(
+        'Enter your email to receive instructions for resetting your password.',
+        style: Theme.of(context)
+            .textTheme
+            .bodyMedium
+            ?.copyWith(color: Colors.white70),
+        textAlign: TextAlign.center,
+      );
+
+  Widget get _form => const ForgetPasswordForm();
+
+  @override
+  Widget build(BuildContext context) {
+    _setupStateListener();
+    return _defaultForgetPasswordView();
+  }
+
+  Widget _defaultForgetPasswordView() {
+    return Stack(
+      children: [
+        _background,
+        Scaffold(
```

Comment:
`Scaffold` should be the root widget imo

Reply:
From my perspective, I think the `Scaffold` is the widget that can divide the screen into basic components, such as app bar, body, bottom bar, etc. It's not forced to be the root widget.

Actually, I tried the approach of having the `Scaffold` as the root widget for this screen. The problem is that I just can put the background widget into the `body` of the `Scaffold`, so the background image cannot cover the whole page:

```dart
Widget _defaultForgetPasswordView() {
    return Stack(
      children: [
        Scaffold(
          backgroundColor: Colors.transparent,
          appBar: _appBar,
          body: Container(
            decoration: BoxDecoration(
              image: DecorationImage(image: _backgroundImage.image),
            ),
            child: BackdropFilter(
              filter: ImageFilter.blur(sigmaX: 10.0, sigmaY: 10.0),
              child: Container(
                color: Colors.black45,
                child: SafeArea(
                  child: Container(
                    padding: const EdgeInsets.symmetric(
                        vertical: 30, horizontal: 24),
                    child: Column(
                      children: [
                        const SizedBox(height: 60),
                        _logo,
                        const SizedBox(height: 20),
                        _intruction(context),
                        const SizedBox(height: 100),
                        _form,
                      ],
                    ),
                  ),
                ),
              ),
            ),
          ),
        ),
      ],
    );
  }
```

<img width="300" alt="Screenshot 2023-02-13 at 17 21 28" src="https://user-images.githubusercontent.com/23162627/218432338-ae40ccb0-7cc8-4b19-a20c-20c1ca090090.png">

Anyway, I think we can try different approaches to implement one UI. It's nice.

Reply:
ah I see, you're right. It's the component to divide the screen into basic components 😄 anw, let's keep it now